packages/bpk-react-utils/src/Portal.js

```diff
@@ -143,9 +143,9 @@ class Portal extends Component {
 
     this.portalElement = document.createElement('div');
     document.body.appendChild(this.portalElement);
-    document.addEventListener('touchstart', this.onDocumentMouseDown, false);
-    document.addEventListener('touchmove', this.onDocumentMouseMove, false);
-    document.addEventListener('touchend', this.onDocumentMouseUp, false);
+    document.addEventListener('touchstart', this.onDocumentMouseDown, { passive: true });
```

Comment:
Without it, browsers have to evaluate the scroll / touch events every time a user scrolls before the browser can actually perform scrolling - if we set passive true, it knows to allow default scroll behaviour and performance will be better